### PR TITLE
Make README.md example compile

### PR DESCRIPTION
**Pull request by sherifabdlnaby**
Readme example wasn't compiling.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,20 +63,20 @@ type ExampleConfig struct {
 // Defines default config option
 var (
     defaultConfig = ExampleConfig{
-		    Counter: 4,
+		    Counter: string(4),
```
Comment (urso) [inline README.md]: better keep `4` here and change the type of counter to `Int`.

**Comment by urso** [comment]
Thanks you!

I left you a comment. Changing the `Counter` paramter to `int` will make the example more correct :)

File: README.md (hunk 1 above)
Comment (sherifabdlnaby) [reply README.md]: I see, I updated the pull-request with counter type `int`

**Comment by sherifabdlnaby** [comment]
@urso 
Thanks, I updated the pull-request accordingly. 🥂

**Comment by urso** [comment]
PR LGTM.

Seems like the CLA webhook is not responding. Double checking with CLA DB I did find you have signed the CLA yesterday.

**Pull request merged**